onadata/apps/api/viewsets/entity_list_viewset.py

```diff
@@ -149,3 +146,18 @@ def create(self, request, *args, **kwargs):
         return Response(
             serializer.data, status=status.HTTP_201_CREATED, headers=headers
         )
+
+    def get_queryset_entities(self, request, entity_list):
+        """Returns queryset for Entities"""
+        search_param = api_settings.SEARCH_PARAM
+        search = request.query_params.get(search_param, "")
+        queryset = Entity.objects.filter(
+            entity_list_id=entity_list.pk, deleted_at__isnull=True
+        )
+
+        if search:
+            queryset = queryset.filter(Q(json__iregex=search) | Q(uuid=search))
+
+        queryset = queryset.order_by("id")
```

Comment:
Is the `order by` necessary? Is it likely to be a performance bottleneck?

Reply:
@ukanga We can choose not to order by default but it ill still be needed if pagination is applied

Reply:
@kelvin-muchiri we fetch this data paginated